### DISCUSSION TITLE
License

### DISCUSSION THREAD

**Discussion by BatmanAndMe** [Rynstick/LBS#3] [General]
Hello @Rynstick, is your library open source and can I use your library for free commercial use?
Thank you for your great work on this library.
Greetings ;)

**Comment by Rynstick** [comment]
Hi @BatmanAndMe  ,

Has mentionned in the description , the library is open source and free to use . It is important to note tho that i am in no way shape or form responsible for the use of the product !

**Reply by BatmanAndMe** [reply]
Hi @Rynstick,
that's nice to hear, thank you very much.
Greetings!
